blocks/blockstore/blockstore.go

```diff
@@ -74,6 +74,10 @@ type blockstore struct {
 }
 
 func (bs *blockstore) Get(k key.Key) (blocks.Block, error) {
+	if k == "" {
+		return nil, ErrNotFound
```

Comment:
Is `ErrNotFound` the right error to return here? I'd think that we want something like "error, invalid key"